composer.json

```
@@ -16,20 +16,19 @@
         }
     ],
     "require": {
-        "php": ">=7.3",
-        "mybuilder/cronos": "~3.0",
-        "doctrine/annotations": "1.*",
-        "symfony/console": "^4.4 || ^5.0 || ^6.0",
-        "symfony/config": "^4.4 || ^5.0 || ^6.0",
-        "symfony/dependency-injection": "^4.4 || ^5.0 || ^6.0",
-        "symfony/framework-bundle": "^4.4 || ^5.0 || ^6.0",
-        "symfony/http-kernel": "^4.4 || ^5.0 || ^6.0",
-        "symfony/property-access": "^4.4 || ^5.0 || ^6.0",
-        "symfony/yaml": "^4.4 || ^5.0 || ^6.0"
+        "php": ">=8.0.2",
+        "mybuilder/cronos": "^3.0 || ^4.0",
+        "doctrine/annotations": "^1.14",
+        "symfony/console": "^4.4.20 || ^5.4 || ^6.0",
```

What was the reason for these being at least 4.4.20?
```suggestion
        "symfony/console": "^4.4 || ^5.4 || ^6.0",
```

---

I changed it for all for conviences and from experience https://github.com/Dukecity/CommandSchedulerBundle/blob/4.x/composer.json
Before 4.4.20 there is a problem i guess it was with simplexml_load_string() https://www.php.net/manual/en/function.simplexml-load-file.php
which cause problems in php > 8.0 or 8.1
I need to run it local to verify.
Could not find infos here https://symfony.com/blog/symfony-4-4-20-released

---

Ah nice find 👍  didn't know about that